src/chesster.js

```diff
@@ -119,6 +119,36 @@ function leagueDMResponse(patterns, responseName) {
 leagueResponse(['captain guidelines'], 'formatCaptainGuidelinesResponse');
 leagueDMResponse(['captains', 'captain list'], 'formatCaptainsResponse');
 
+/* game nomination */
+chesster.hears({
+    middleware: [ slack.requiresLeague ],
+    patterns: [ 'nomination' ],
+    messageTypes: [ 'direct_message' ]
+},
+function(bot, message) {
+    if (!message.league) {
+        return;
+    }
+    var heltourOptions = message.league.options.heltour;
+    if (!heltourOptions) {
+        winston.error("{} league doesn't have heltour options!?".format(message.league.options.name));
+        return;
+    }
+
+    var speaker = users.getByNameOrID(message.user);
+    heltour.getPrivateURL(
+        heltourOptions,
+        'nominate', //hardcoding this for now
+        speaker.name
+    ).then(function(getPrivateURLResult){
+        var json = getPrivateURLResult.json;
```

Comment:
If you return response['json'] from the previous method then you don't need this line here, you get the json object directly. Makes this code a bit more simple.